index.rst

```diff
@@ -42,20 +42,163 @@
 
 .. sectnum::
 
-.. TODO: Delete the note below before merging new content to the master branch.
+.. _introduction:
+
+Introduction
+============
+
+At the summit it is sometimes desirable for instrument data to be analyzed rapidly after data acquisition in order to get quick feedback of data quality.
+The different use cases for processing data are described in :cite:`DMTN-111` and in this document we focus on data processing directly driven by data acquisition.
+
+Data processing must be triggered in some way with enough information to be able to work out which data to process and which analysis pipeline to use.
+The unique identifier for every observation is the "Observation Identifier" (aka "obsid" aka "image name") that is guaranteed by the observatory systems to be unique for every observation regardless of telescope (main telescope vs AuxTel), instrument, or which controller was involved to take the data.
+This obsid is issued by the camera control system whenever data are acquired.
+Additionally, there is a "grouping" concept that can be used by scripts to indicate that observations taken with the same group identifier are related.[*]_
```

Comment:
This footnote doesn't link properly.  See https://developer.lsst.io/restructuredtext/style.html#footnotes
```suggestion
Additionally, there is a "grouping" concept that can be used by scripts to indicate that observations taken with the same group identifier are related.\ [*]_
```